index-template-mapping.json

```diff
@@ -8,15 +8,17 @@
     }
   },
   "mappings": {
+    "index": {
```

Comment:
I don't understand this? I cannot find in the ES docs that something the like would be supported, where do you have this from?

Reply:
May be I got the subtle difference. 
@vanthome could you confirm that `index-template-mapping.json` is for logstash? 
Because I'm pretty sure that this json does not work if you use it directly into kibana. 
Just try into kibana -> Dev Tools 

```
PUT _template/template_1
{
  "index_patterns": "logs-*",
  "settings": {
    "number_of_shards": 1,
    "number_of_replicas": 0,
    "index": {
      "refresh_interval": "5s"
    }
  },
  "mappings": {
    "_source": { "enabled": true },
    "properties": {
      "@timestamp": { "type": "date" },
      "@version": { "type": "keyword" },
      "message": { "type": "text", "index": true },
      "severity": { "type": "keyword", "index": true },
      "fields": {
        "dynamic": true,
        "properties": { }
      }
    }
  }
} 

```

Reply:
Yes, the question is why you have added `index { }`?
Yes, this is for Kibana and it currently works fine

Reply:
Another update. I've tried your `index-template-mapping.json` with a newer version of Elasticsearch (7.6.1) and it works. So it's not working with older ES versions. And, as you know firsthand, there is no compatibility between new mapping template and older. I think we could close the PR, there is nothing to fix.

Reply:
@vanthome, and last thing, thanks for this valuable piece of opensource software.

Reply:
thx currently we use 7.5.2